Library/Homebrew/blacklist.rb

```diff
@@ -83,5 +83,8 @@ def blacklisted?(name)
     A binary installer is available:
       https://www.haskell.org/platform/mac.html
     EOS
+  when "mysqldump-secure" then <<-EOS.undent
+    The creator of mysqldump-secure tried to game our popularity metrics.
```

Comment:
@cytopia Do you admit to signing up to new GitHub accounts for this? It's worth noting that's in violation of GitHub's terms of service.